node/src/chain_spec.rs

```diff
@@ -139,7 +140,8 @@ pub fn development_config() -> Result<ChainSpec, String> {
 }
 
 pub fn local_testnet_config() -> Result<ChainSpec, String> {
-    let wasm_binary = WASM_BINARY.ok_or("Development wasm binary not available".to_string())?;
+    let wasm_binary =
+        WASM_BINARY.ok_or_else(|| "Development wasm binary not available".to_string())?;
```

Comment:
What is the difference between `.ok_or(smth)` and `ok_or_else(|| smth)`? What is the rule to choose between two?